app.py

```
@@ -61,8 +61,29 @@ def events():
-@app.route('/apply')
+@app.route('/apply', methods=['GET', 'POST'])
 def apply():
+    if request.method == 'POST':
+        name = request.form.get('name')
+        email = request.form.get('email')
+        phone = request.form.get('phone')
+        # ... other form fields ...
+
+        try:
+            # Update your database insert to include phone
+            data = supabase.table('registrations').insert({
```

Ruff (0.8.2)

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)